### PR TITLE
Fever Blossom changes, return of the bock, new canaanite preacher, more degeneracy nuked for Super Earth

### PR DESCRIPTION
**Pull request by biotear**
## About The Pull Request
Uncomments ballistic bock, changes the fermented form of fever blossom to a new drink called fever dream, changes fever blossom description to be less horny, and adds new reagents to fever blossom to replace the removed aphrodisiacs it had before. Doubles as a way to get med-x for battle brew that doesn't require CBT fermichem or tribal girl pukies filtering.

Now also nukes the furshit bottle from one of the preacher loadouts, and gives them a new canaanite loadout with 10 M1911s so they can larp as the burned man. All as Joseph Smith intended...

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: fever dream
add: new canaanite loadout for preacher
tweak: ballistic bock
tweak: fever blossom
tweak: replaces the wataur bottle for preacher samaritan with a third water bottle.
/:cl:

### CODE REVIEW

**Comment by Aurrain** [comment]
bock

**Comment by biotear** [comment]
> bock

ZAX, play the Robot Chicken song.

Anyway not sure what's causing the linters.

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -1410,26 +1411,29 @@ Raider
...
 /datum/outfit/loadout/cleanser
 	name = "Cleanser"
 	backpack_contents = list(
-		/obj/item/grenade/homemade/coffeepotbomb = 2,	//This is funny. I swear guys.
-		///obj/item/gun/ballistic/revolver/m29 = 1,		//Moved here for more *variety*
-		///obj/item/ammo_box/m44 = 2
+		/obj/item/grenade/homemade/coffeepotbomb = 2	//This is funny. I swear guys.
+	)
+
+	/datum/outfit/loadout/newcanaan
```
Comment (Aurrain) [inline code/modules/jobs/job_types/wasteland.dm]: You need to slide this back a bit, it's a nested absolute path when you press TAB on this line like this.

**Review by Aurrain — APPROVED** [review]
Works now.

File: code/modules/jobs/job_types/wasteland.dm (hunk 1 above)
Comment (biotear) [reply code/modules/jobs/job_types/wasteland.dm]: Ye, I am blind.

**Comment by biotear** [comment]
if all checks pass, this is ready to be merged.

**Pull request merged**